.github/workflows/workflow-run-replay-verify.yaml

```diff
@@ -80,7 +80,15 @@ jobs:
   replay-verify:
     timeout-minutes: ${{ inputs.TIMEOUT_MINUTES || 720 }}
     runs-on: ${{ inputs.RUNS_ON }}
+    strategy:
+        matrix:
+          number: [0, 1, 2, 3, 4, 5, 6, 7] # runner number
```

Comment:
nit: if possible, it would be nice to have some top-level config like NUM_RUNNERS or something. Otherwise, someone who wants to update this (do you expect this number of runners to be tuned?) has to do it in two places

Reply:
not found an easy way to do this. But if this turns out to be a problem. I will investigate how to auto generate the array here